.hooks/pre-push

```diff
@@ -2,31 +2,22 @@
 # exit on error
 set -e
 
-modifiedPhpFiles="$(git diff master...HEAD --diff-filter=d --name-only '**/*.php')"
-if [ ${#modifiedPhpFiles} -gt 0 ]; then
-    printf "\n⏳ composer lint\n"
-    PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none --config=.php-cs-fixer.php ${modifiedPhpFiles}
+printf "\n⏳ composer lint\n"
+PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none --config=.php-cs-fixer.php
 
-    printf "\n⏳ composer analyse\n"
-    vendor/bin/phpstan analyse --memory-limit 512M
+printf "\n⏳ composer analyse\n"
+vendor/bin/phpstan analyse --memory-limit 512M
 
-    printf "\n⏳ composer deptrac\n"
-    vendor/bin/deptrac analyse
+printf "\n⏳ composer deptrac\n"
+vendor/bin/deptrac analyse
 
-    printf "\n⏳ composer test\n"
-    vendor/bin/phpunit
```

Comment:
oh my, just relaized that the phpunit step will be missing 😅

Reply:
can you please add it back?

Reply:
nevermind, noticed it was added at the end, all good 🤦

Reply:
It's there. I just moved it to the end. It's the slowest step, so that lets everything else have a chance to fail quickly first.

Reply:
perfect, thanks!